app/src/main/res/values/strings.xml

```diff
@@ -787,7 +787,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_leaf_type_needles">Needles</string>
     <string name="quest_leaf_type_broadleaved">Broadleaved</string>
     <string name="quest_leaf_type_mixed">Both are present</string>
-    <string name="create_new_note_hint">Best write the note in the locally spoken language or otherwise in English.</string>
+    <string name="create_new_note_hint">Ideally, write the note in the local language, or otherwise in English.</string>
```

Comment:
Pedantically some places could have more than one local language, see e.g. https://en.wikipedia.org/wiki/Multilingualism_in_Luxembourg . I don't know if we care enough to say?
```suggestion
    <string name="create_new_note_hint">Ideally, write the note in a local language, or otherwise in English.</string>
```

I suspect most French people say in Germany have a good enough grasp of English to write it in English, but they are probably one of the few edge cases.

Reply:
I asked one more extra native speaker and they recommended "the" version

Reply:
> I asked one more extra native speaker and they recommended "the" version

I don't think this is a "native speaker" type of problem, @peternewman's example of Luxembourg is great, the language used is dependent on the situation, not the geographical area. I'd therefore go with "a local language".

Reply:
I mentioned to them also issue of multiple local languages.

(I do not really feel qualified to judge here)